clang/utils/TableGen/ClangAttrEmitter.cpp

```diff
@@ -3860,6 +3860,11 @@ void EmitClangAttrHasAttrImpl(const RecordKeeper &Records, raw_ostream &OS) {
         Pragma.emplace_back(R, SI);
       else if (Variety == "HLSLAnnotation")
         HLSLAnnotation.emplace_back(R, SI);
+      else if (Variety == "AS_Annotation") {
+        // We should not be code gening anything with a C++26
```

Comment:
Yeah, we should probably diagnose this, an annotation having a spelling is ill-formed.  I do wonder if we should instead just omit the 'spellings' in Attr.td and allow that/diagnose on 'has spellings'.